js/assessments/countWords.js

```diff
@@ -0,0 +1,87 @@
+var countWords = require( "../stringProcessing/countWords.js" );
+
+var Jed = require( "jed" );
+
+var constructi18n = function() {
```

Comment:
Try and see if the i18n translating can be retrieved from elsewhere. Redefining this within every assessment will make maintenance a lot more cumbersome.